network-api/networkapi/buyersguide/templates/buyersguide/bg_base.html

```diff
@@ -0,0 +1,112 @@
+{% extends "pages/base.html" %}
+
+{% load bg_nav_tags localization i18n static wagtailroutablepage_tags %}
+
+{% get_current_language as lang_code %}
+
+{% block page_title %}{% if pageTitle %}{{ pageTitle }}{% else %}Mozilla - {% blocktrans context "This can be localized. This is a reference to the “*batteries not included” mention on toys." %}*privacy not included{% endblocktrans %}{% endif %}{% endblock %}
+
+
+{# TODO: consider using a different ga_identifier? #}
+
+
+{% block social_metadata %}
+  <meta property="og:type" content="website">
+  <meta property="og:locale" content="{{ lang_code|to_opengraph_locale }}">
+  <meta property="og:title" content="{% trans "*Privacy Not Included: A Buyer’s Guide for Connected Products" context "*Privacy Not Included can be localized. This is a reference to the “*batteries not included” mention on toys." %}">
+  <meta property="wagtail:language" content="{{ lang_code }}">
+{% endblock %}
+
+
+{% block wagtail_metadata %}
+  <meta property="og:description" content="{% trans "Smart home gadgets, fitness trackers, toys and more, rated for their privacy & security" %}">
+  <meta property="og:image" content="{% static "_images/buyers-guide/evergreen-social.jpg" %}">
+  <meta property="og:image:type" content="image/jpg">
+  <meta property="og:image:width" content="1200">
+  <meta property="og:image:height" content="628">
+  <meta property="og:url" content="{{request.scheme}}://{{request.get_host}}{{request.path}}">
```

Comment:
we'll have to remember to replace this with the wagtail metadata block instead - if we don't have an issue open for that yet, we should add that to the epic.